src/components/policy/policy_external/include/policy/policy_table/types.h

```diff
@@ -229,10 +229,35 @@ struct RpcParameters : CompositeType {
   bool Validate() const;
 };
 
+struct ExternalConsentEntity : CompositeType {
+ public:
+  typedef Integer<int32_t, INT32_MIN, INT32_MAX> EntityInt;
+  EntityInt entity_type;
+  EntityInt entity_id;
+
+ public:
+  ExternalConsentEntity();
+  explicit ExternalConsentEntity(const Json::Value* value__);
+  ExternalConsentEntity(const int32_t type, const int32_t id);
+  Json::Value ToJsonValue() const;
+  bool operator==(const ExternalConsentEntity& rhs) const;
+  bool is_valid() const;
```

Comment:
@LuxoftAKutsan Correct me if I am wrong, but quick reference to the base class in [rpc_base.h](sdl_core/src/components/rpc_base/include/rpc_base/rpc_base.h) shows that all applicable methods have been marked already.